sdks/python/apache_beam/metrics/metric_test.py

```diff
@@ -172,10 +172,13 @@ def process(self, element):
     # Verify user distribution counter.
     metric_results = res.metrics().query()
     matcher = MetricResultMatcher(
-        namespace='apache_beam.metrics.metric_test.SomeDoFn',
+        step='ApplyPardo',
+        namespace=hc.contains_string('SomeDoFn'),
         name='element_dist',
         committed=DistributionMatcher(
-            sum_value=10, count_value=4, min_value=1, max_value=4))
+            sum_value=10, count_value=4,
```

Comment:
why would min value be >= 0 but the other values be exact?

Reply:
b/c after repeating it multiple times, streaming side gives consistent outcome on these exact numbers. 

After another thought though, given how much uncertainties there are on streaming side, it is really not necessary to put exact number here. And if we do really want to check exact number, there is alternative way to add that separately. 

Updated by making all other threes to be 'greater_than(0)'.  PTAL.